### ISSUE TITLE
Jupyterlab 2.0 support

### ISSUE THREAD

**Issue by wookayin** [chaoleili/jupyterlab_tensorboard#20]
Jupyterlab 2.0 has been released recently, but it doesn't work with Jupyterlab 2.0.

As of 2020/3/8:

```
❯❯❯ jupyter labextension install jupyterlab_tensorboard

An error occured.
ValueError: The extension "jupyterlab_tensorboard" does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab              Extension      Package
>=2.0.2 <2.1.0          >=1.0.0 <2.0.0 @jupyterlab/application
>=2.0.2 <2.1.0          >=1.0.0 <2.0.0 @jupyterlab/apputils
>=4.0.2 <4.1.0          >=3.0.0 <4.0.0 @jupyterlab/coreutils
>=2.0.2 <2.1.0          >=1.0.0 <2.0.0 @jupyterlab/launcher
>=5.0.2 <5.1.0          >=4.0.0 <5.0.0 @jupyterlab/services
```

```
❯❯❯ pip freeze | grep jupyterlab
jupyterlab==2.0.1
jupyterlab-server==1.0.6
```

**Comment by MichaelJiangPlus** [comment]
https://github.com/chaoleili/jupyterlab_tensorboard/pull/21    You can try to support jupyterlab 2.0 by modifying the dependencies of jupyterlab-tensorboard

**Comment by consideRatio** [comment]
I think this is closed by #21 and the fresh release of version 0.2.0.
